src/textual/widgets/_label.py

```diff
@@ -5,3 +5,11 @@
 
 class Label(Static):
     """A simple label widget for displaying text-oriented renderables."""
+
+    DEFAULT_CSS = """
+    Static {
```

Comment:
Why didn't you write the rule as `Label { ... }`?

Reply:
Too early in the morning and not enough coffee. 😛

Reply:
Fixed. Good catch. :-)